src/PixiEditor/ViewModels/SubViewModels/LayersViewModel.cs

```diff
@@ -131,7 +132,7 @@ public bool CanCreateNewMember()
     public void ToggleLockTransparency()
     {
         var member = Owner.DocumentManagerSubViewModel.ActiveDocument?.SelectedStructureMember;
-        if (member is not LayerViewModel layerVm)
+        if (member is not ImageLayerNodeViewModel layerVm)
```

Comment:
I think checking against `ImageLayerNodeViewModel` is justified here, since I don't think other layer types would use lock transparency. Gj